package.json

```diff
@@ -43,7 +43,7 @@
     "throttleit": "^1.0.0",
     "unlimited": "^1.1.0",
     "webtorrent": "0.x",
-    "webtorrent-app": "feross/webtorrent-app",
+    "webtorrent-desktop": "git+ssh://git@github.com/feross/webtorrent-desktop.git",
```

Comment:
This probably stopped working when we renamed the repo. Looks like you changed it back and it works fine, right?